MastodonSDK/Sources/MastodonUI/Scene/ComposeContent/Toolbar/ComposeContentToolbarView+ViewModel.swift

```diff
@@ -83,6 +91,8 @@ extension ComposeContentToolbarView.ViewModel {
                 return Asset.Scene.Compose.chatWarning.image.withRenderingMode(.alwaysTemplate)
             case .visibility:
                 return Asset.Scene.Compose.earth.image.withRenderingMode(.alwaysTemplate)
+            case .language:
+                fatalError()
```

Comment:
Tbh: I have a bad gut feeling with these `fatalError()`, but I don't have an idea, how to do it better in this case. From my perspective we must have a `UIImage` and can't use a `UIImage?` here, right?

Reply:
Yeah. An alternative that I’d be happy to do here or in a separate PR would be to remove the computed variables here and instead define the appropriate properties in the view…

```swift
case .attachment:
  Menu {
    // ...
  } label: {
    ComposeContentToolbarLabel(image: Asset.Scene.Compose.media)
  }.disabled(!viewModel.isAttachmentButtonEnabled)
case .poll:
  Button {
    // ...
  } label: {
    ComposeContentToolbarLabel(image: viewModel.isPollActive ? Asset.Scene.Compose.pollFill : Asset.Scene.Compose.poll)
  }.disabled(!viewModel.isPollButtonEnabled)
```

Reply:
Maybe for now you can just add a short message to the `fatalError()`, why this should never ever happen? For the bigger refactoring I'd suggest to use its own PR. What do you think?

Reply:
Sure!

Reply:
Done. Will open the other PR shortly!